apps/communications/dialer/elements/call-info-view.html

```diff
@@ -14,6 +14,8 @@ <h1 id="call-info-title"></h1>
 
       <div id="call-durations">
       </div>
+
+      <a id="show-more" data-l10n-id="showMore"></a>
```

Comment:
You will have to move this into ```<div id="call-durations"></div>```. Then you can use this selector to match the case I was talking about before:
```css
#show-more ~ li:nth-child(11) ~ #show-more {
  display: none;
}
```